src/ocean/text/convert/Formatter.d

```diff
@@ -36,15 +36,39 @@ version (UnitTest)
 
 /*******************************************************************************
 
-    Type of 'sink' that can be passed to `format`, matches `Layout.SizeSink`
+    Type of 'sink' that can be passed to `format`, that will just format a
+    string into the provided sink.
 
 *******************************************************************************/
 
+public alias void delegate(cstring) FormatterSink;
+
+/// Deprecated as the `size_t` return is not used. Use `FormatterSink` instead.
+deprecated("Use FormatterSink instead, Sink's return value is never used")
 public alias size_t delegate(cstring) Sink;
 
 
 /*******************************************************************************
 
+    Internal sink type that wraps the user-provided one and takes care
+    of cropping and width
+
+    This sink expects to receive a full element as the first parameter,
+    in other words, the full chunk of text that needs a fixed size.
+    This is why one cannot format a whole aggregate (struct, arrays etc.), but
+    only individual elements.
+
+*******************************************************************************/
+
+private alias size_t delegate(cstring, ref Const!(FormatInfo)) PElementSink;
+
+/// This was accidentally made private so needs to be deprecated
+deprecated("This alias shouldn't be used from outside the Formatter")
```

Comment:
Not really. The sink type needed to be updated to have `void` return type instead of `size_t`.
I just forgot to `git add` it in a previous version of this commit :blush: